infra/network-isolation.bicep

```diff
@@ -75,83 +130,82 @@ module cognitiveservicesDnsZone './core/networking/private-dns-zones.bicep' = {
 module openaiDnsZone './core/networking/private-dns-zones.bicep' = {
   name: 'openai-dnzones'
   params: {
-    dnsZoneName: 'privatelink.openai.azure.com' 
+    dnsZoneName: 'privatelink.openai.azure.com'
     tags: tags
     virtualNetworkName: vnet.outputs.name
   }
 }
 
-module searchDnsZone './core/networking/private-dns-zones.bicep' =  {
+module searchDnsZone './core/networking/private-dns-zones.bicep' = {
   name: 'searchs-dnzones'
   params: {
-    dnsZoneName: 'privatelink.search.windows.net' 
+    dnsZoneName: 'privatelink.search.windows.net'
```

Comment:
Will these values be static across all clouds?  i.e. azure global.  If not, then please default and add ability to override.

Reply:
Yes, they should be. Also they should be in environmentData.suffixes but not all of them are, so they're hard coded

Reply:
We'll raise an issue on the bicep repo